src/vc_util/src/lib.rs

```diff
@@ -83,20 +83,21 @@ pub fn did_for_principal(principal: Principal) -> String {
 pub fn verify_id_alias_credential_jws(
     credential_jws: &str,
     alias_tuple: &AliasTuple,
-    canister_sig_pk: &CanisterSigPublicKey,
+    canister_id: &Principal,
     root_pk_raw: &[u8],
 ) -> Result<(), CredentialVerificationError> {
-    let claims = verify_credential_jws(credential_jws, canister_sig_pk, root_pk_raw)
+    let claims = verify_credential_jws_with_canister_id(credential_jws, canister_id, root_pk_raw)
         .map_err(CredentialVerificationError::InvalidJws)?;
     validate_id_alias_claims(claims, alias_tuple)
         .map_err(CredentialVerificationError::InvalidClaims)
 }
 
-/// Verifies the specified JWS credential cryptographically.
+/// Verifies the specified JWS credential cryptographically and checks that the signature was
+/// created by the provided canister.
 /// DOES NOT perform semantic validation of the claims in the credential.
-pub fn verify_credential_jws(
+pub fn verify_credential_jws_with_canister_id(
     credential_jws: &str,
-    canister_sig_pk: &CanisterSigPublicKey,
+    canister_id: &Principal,
```

Comment:
nit: maybe rename to `signing_canister_id`, so that it is clear which canister id is meant?